lib/Interpreter/LookupHelper.cpp

```diff
@@ -1910,6 +1911,10 @@ namespace cling {
     assert(Type && "Type cannot be null");
     const Transaction*& Cache = m_Interpreter->getStdStringTransaction();
     if (!Cache || !m_StringTy[kStdString]) {
+      // getStringType can be called multiple times with Cache being null, and
+      // the local cache should be discarded when that occurs.
+      if (!Cache) ::memset(m_StringTy, 0, sizeof(m_StringTy));
```

Comment:
Can we leave this line (and this comment) but rephrase it as
```
m_StringTy = {};
```
That should be fine, even if leaving the in-class initializer, or am I overlooking something?